src/pages/Dashboard/Families/FamilyDetail/FamilyDetail.tsx

```diff
@@ -1,26 +1,40 @@
 import { DashboardCard } from '@camp/components';
 import { messages } from '@camp/messages';
+import { useMatches } from '@camp/router';
+import { useEffect, useState } from 'react';
+
+import { useDraftFamilyDetailQuery } from '../../../../data-layer/operations/__generated__/typesAndHooks';
 
 export const FamilyDetail = () => {
   const t = messages.familyDetail.familyFields;
+  const [familyId, setFamilyId] = useState('');
+  const matches = useMatches();
+
+  useEffect(() => {
+    matches.map(match => setFamilyId(match.params.familyId));
+  }, [familyId, matches]);
+
+  const { data } = useDraftFamilyDetailQuery({
+    variables: { id: familyId },
+  });
```

Comment:
do we need familyId to be a state?
```suggestion
 const [match] = useMatches();
  const familyId = match!.params.familyId;

  const { data } = useDraftFamilyDetailQuery({
    variables: { id: familyId },
  });
```